frontend/javascripts/oxalis/controller/scene_controller.ts

```diff
@@ -430,23 +435,42 @@ class SceneController {
     this.rootNode.add(this.userBoundingBoxGroup);
   }
 
+  private applyTransformToGroup(transform: Transform, group: THREE.Group | CustomLOD) {
+    if (transform?.affineMatrix) {
```

Comment:
why the `?`? isn't `Transform` always defined? and if it can be null, shouldn't `group.matrix` be set to identity or something like that?